extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -181,8 +181,8 @@ class AmpLightbox extends AMP.BaseElement {
     this.action_ = Services.actionServiceForDoc(this.element);
     this.maybeSetTransparentBody_();
 
-    this.registerDefaultAction(unused => this.open_(), 'open');
-    this.registerAction('close', this.close.bind(this));
+    this.registerDefaultAction(i => this.open_(i.trust), 'open');
```

Comment:
micronit: destructure?

```js
({trust}) => this.open_(trust)
```

Reply:
IMO dot syntax is more readable here, especially for a single property. :)